qemu/tests/cfg/nested_libguestfs_unittest.cfg

```diff
@@ -0,0 +1,25 @@
+- nested_libguestfs_unittest:
+    type = nested_libguestfs_unittest
+    virt_test_type = qemu
+    no RHEL.6, RHEL.7
+    no Host_RHEL.m6, Host_RHEL.m7
+    required_qemu = [3.0, )
+    start_vm = no
+    unittest_timeout = 600
+    only ppc64le, x86_64
+    ppc64le:
+        nested_symbol = "kvm_hv"
+    x86_64:
+        nested_symbol = "kvm_intel"
```

Comment:
Currently, x86 AMD is supported on RHEL 8.2, but not enabled by default. Maybe we need to consider support AMD in the future. FYI.

Reply:
OK, updated.